example/spec/parallel/testAPI/sqsRuleSpec.js

```diff
@@ -222,7 +224,7 @@ describe('The SQS rule', () => {
     });
 
     afterAll(async () => {
-      const key = getS3KeyForArchivedMessage(config.stackName, messageId);
+      const key = getS3KeyForArchivedMessage(config.stackName, messageId, queueName);
```

Comment:
It seems like we could assign `key = getS3KeyForArchivedMessage(config.stackName, messageId, queueName);` in the `beforeAll` and not have to call this function twice

Reply:
[`9cdb9d2` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/9cdb9d2ca4368869d4f4c37ef224b06788f074cf)